src/services/queue/sendInitiateDataTransferMessage.ts

```diff
@@ -0,0 +1,11 @@
+import { DataRequestParams } from '../../types/dataRequestParams'
+import { getEnv } from '../../utils/helpers'
+import { sendSqsMessage } from './sendSqsMessage'
+export const sendInitiateDataTransferMessage = async (
+  dataRequestParams: DataRequestParams
+) => {
+  await sendSqsMessage(
```

Comment:
Do we need to call await here or can we fire and forget?

Would it make more sense if this function returned the promise so other functions could wait on it's completion?